src/gocart_seas_default_mod.F90

```diff
@@ -17,81 +17,77 @@ module gocart_seas_default_mod
-                                            tsk,            &
-                                            area
-
-     REAL(kind=kind_chem), DIMENSION( num_chem ),                 &
-           INTENT(INOUT ) ::                                   chem_arr
-     REAL(kind=kind_chem), DIMENSION( num_emis_seas),                    &
-           INTENT(OUT   ) ::                                   emis_seas
-
-!
-!
-! local variables
-!
+  subroutine gocart_seas_default(ktau,dt,u_phy,v_phy,chem_arr,dz8w,u10,         &
+       v10,delp,tsk,area,emis_seas)
+    
+    ! Input Variables 
+    ! ---------------
+    INTEGER, INTENT(IN) :: ktau
+    REAL(kind=kind_chem), INTENT(IN) :: dt
+    REAL(kind=kind_chem), INTENT(IN) :: u_phy
+    REAL(kind=kind_chem), INTENT(IN) :: v_phy
+    REAL(kind=kind_chem), INTENT(IN) :: dz8w
+    REAL(kind=kind_chem), INTENT(IN) :: u10
+    REAL(kind=kind_chem), INTENT(IN) :: v10
+    REAL(kind=kind_chem), INTENT(IN) :: delp
+    REAL(kind=kind_chem), INTENT(IN) :: tsk ! this isn't used anywhere
+    REAL(kind=kind_chem), INTENT(IN) :: area
+    
+    ! Output Variables 
+    ! ----------------
+    REAL(kind=kind_chem), DIMENSION( num_chem ), INTENT(INOUT ) :: chem_arr
+    REAL(kind=kind_chem), DIMENSION( num_emis_seas), INTENT(OUT) :: emis_seas
+    
+    ! local variables
+    ! ---------------
     integer :: ipr,n,rc,ilwi
-    real(kind=kind_chem) :: fsstemis, memissions, nemissions, tskin_c, ws10m
-    real(kind=kind_chem) :: dummylon, fgridefficiency,deep_lakes_mask
-    real(kind=kind_chem), DIMENSION (number_ss_bins) :: tc,bems
-    real(kind=kind_chem) :: w10m,airmas,tskin
-    real(kind=kind_chem) :: dxy
-
+    real(kind=kind_chem) :: fsstemis
+    real(kind=kind_chem) :: memissions
+    real(kind=kind_chem) :: nemissions
+    real(kind=kind_chem) :: ws10m
+    real(kind=kind_chem) :: dummylon
+    real(kind=kind_chem) :: fgridefficiency
+    real(kind=kind_chem) :: deep_lakes_mask
+    real(kind=kind_chem) :: w10m 
+    real(kind=kind_chem) :: airmas
+    real(kind=kind_chem), DIMENSION (number_ss_bins) :: tc
+    real(kind=kind_chem), DIMENSION (number_ss_bins) :: bems
+    
     real(kind=kind_chem) :: airmas1
-    real(kind=kind_chem), dimension(number_ss_bins) :: tc1
-    real(kind=kind_chem), dimension(number_ss_bins) :: bems1
-    real(kind=kind_chem) :: one
 
-! local parameters
-!
+    ! local parameters
+    ! ----------------
     real(kind=kind_chem), parameter :: conver  = 1.e-9_kind_chem
     real(kind=kind_chem), parameter :: converi = 1.e+9_kind_chem
-!
-    one = 1.0
 
-! -- original GOCART sea salt scheme
+
+    ! -- original GOCART sea salt scheme
     ! -- compute auxiliary variables
+
+    ! Calcualte 10m from first model layer if first layer is low
     if (dz8w < 12.) then
       ws10m = sqrt(u_phy*u_phy+v_phy*v_phy)
     else
       ws10m = sqrt(u10*u10+v10*v10)
     end if
 
-    ilwi=0
+    ilwi=0 ! this is always zero is it needed? 
```

Comment:
Indeed seems like it would be better to eliminate it and eliminate that corresponding else branch.